CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, age, or religion.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
```

Comment:
>If you disagree, feel free to open a separate issue for this so that those who want to make the case that a CoC is actively harmful can do so without disrupting existing discussion over the wording of the CoC we already have.

I don't think it's harmful at all, if people want to have a CoC, they should have one, no matter how pointless it is. What I take issue with is people wasting developer time with frivolous issues that simply don't exist.

You also pretty much hit the nail on the head with 

>Well, it's actually happened a few times here since this drama started, but I don't think I've ever seen any instances before that.

These same people who go around calling everything "problematic" and "offensive" are the ones causing the problem in the first place by bringing their gender politics into a place it's not needed or wanted.